examples/plotting/README.md

```diff
@@ -12,10 +14,6 @@ associated with your data. A simple, but typical complete example of this API lo
 
     show(p)
 
-There are three subdirectories:
-
-* `file` examples that output to static HTML files
-
-* `notebook` examples that display inline in [Jupyter](http://jupyter.org) notebooks
+## Structure
```

Comment:
I don't think an entire new subsection is needed just for one sentence, I would delete this and just leave the sentence below on its own.